src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(ProductService.class);
```

Comment:
According to the checklist, you should use `@Inject` annotation for injecting dependencies. Directly calling `getInstance` method from `Injector` in the `Main` class is not the best practice for dependency injection. Instead, consider using field injection with `@Inject` annotation in appropriate places.

Reply:
ok, done

Reply:
I had to remove the @Inject annotation. I think I don't understand what exactly should I do...